source/opt/folding_rules.cpp

```diff
@@ -2484,6 +2600,8 @@ void FoldingRules::AddFoldingRules() {
   rules_[SpvOpSNegate].push_back(MergeNegateMulDivArithmetic());
   rules_[SpvOpSNegate].push_back(MergeNegateAddSubArithmetic());
 
+  rules_[SpvOpBitcast].push_back(BitCastScalarOrVector());
+
```

Comment:
We've been trying to keep these in alphabetical order, so the OpBitcast should be further up.  That got messed up with the image instructions, but please move this one anyway.  Thanks.